pkg/minikube/out/out.go

```diff
@@ -126,12 +126,12 @@ func boxedCommon(printFunc func(format string, a ...interface{}), cfg box.Config
 
 // Boxed writes a stylized and templated message in a box to stdout using the default style config
 func Boxed(format string, a ...V) {
-	boxedCommon(String, defaultBoxCfg, "", format, a...)
+	boxedCommon(Stringf, defaultBoxCfg, "", format, a...)
```

Comment:
hmm, looking at the boxedCommon() func, it accepts printFunc, so not sure if trailing "\n" might be needed (as Stringf uses Outputf, that in turn, uses fmt.Fprintf)

Reply:
I renamed the `String` func to `Stringf` but none of the logic changed, so this should be identical to before.

Reply:
hmm, i think it might've changed:

renamed String to Stringf now uses Outputf that uses fmt.Fprintf:
https://github.com/kubernetes/minikube/pull/19447/files#diff-b03808f35b53bc429e59e95cf4d9356db35223bff1d5c9fdebf4693b1746a088R207

and new String uses Output that uses fmt.Fprint:
https://github.com/kubernetes/minikube/pull/19447/files#diff-b03808f35b53bc429e59e95cf4d9356db35223bff1d5c9fdebf4693b1746a088R183

Reply:
i might be wrong tho - looks like fmt.Fprint does not add new line automatically:
https://pkg.go.dev/fmt#example-Fprint

Reply:
yep, https://pkg.go.dev/fmt#Fprintln does but not Fprint
sorry about the confusion

Reply:
I renamed `String` -> `Stringf`, `Output` -> `Outputf`, and `Err` -> `Errf`. Then added new versions of `String`, `Output`, and `Err` that don't support formatting directives. So calling `Stringf` which calls `Outputf` should be identical as previously calling `String`.

Before, `Output` used `fmt.Fprintf`:
```
func Output(file fdWriter, format string, a ...interface{}) {
	_, err := fmt.Fprintf(file, format, a...)
	if err != nil {
		klog.Errorf("Fprintf failed: %v", err)
	}
}
```

Reply:
now it actually has more sense, but reversing to the previous setup was not obvious at first :)
thanks for explaining

Reply:
Thanks for the review!